index.html

```diff
@@ -96,11 +96,11 @@
                 var mm = today.getMonth()+1; //January is 0!
                 var yyyy = today.getFullYear();
 
-                if (dd == 14 && mm == 3) {
+                if (dd == 29 && mm == 3) {
                     var answer = 'Holi kal thi. Kal thi Holi!';
-                } else if (dd == 12 && mm == 3) {
+                } else if (dd == 27 && mm == 3) {
                     var answer = 'Holi kal hai. Kal hai Holi!';
-                } else if (dd == 13 && mm == 3) {
+                } else if (dd == 28 && mm == 3) {
```

Comment:
Holi is on 29th March technically, 28th is Holika Dahana [1]

[1] https://calendar.google.com/calendar/u/0/r/month/2021/3/1?pli=1